src/Streams.ts

```diff
@@ -0,0 +1,337 @@
+import { Buffer } from 'buffer'
+import { nextTick } from 'process'
+import { Readable, Writable } from 'stream'
```

Comment:
Change this to `import { Readable, Writable } from 'readable-stream';`. This is portable stream library.

Reply:
Done. This can only be done with `const { Readable, Writable } = require('readable-stream')`